tests/pytests/l3_dump_test.py

```diff
@@ -216,7 +216,7 @@ def test_cc_test_dump_log_entries():
     make_rv = exec_make(['make', 'all-cc-tests'],
                         { "BUILD_VERBOSE": "1",
                           "CC": "g++", "CXX": "g++", "LD": "g++",
-                          "L3_LOC_ENABLED": L3_LOC_DEFAULT} )
```

Comment:
This, and L191 above, were latent test-case bugs left-over from previous work.

These cases are supposed to test w/o LOC-encoding and supplying this env-var was incorrectly generate LOC-encoding as part of logging.